### ISSUE TITLE
after update to newest firmware, temeprature jumps around

### ISSUE THREAD

**Issue by TylonHH** [MarlinFirmware/Marlin#5776]
Hi,
In the past I used an old modified firmware from an user wich works fine. As I red all the new stuff you provide in the newest release, I gave it a try.
So I downloaded from here and modified it to my coreXY printer.
Then I noticed in S3D that the thermistor from heatbed jumps from -43 to over 700°C "randomly". On my 12864 full graph Display I dont notice that, except that the bed temperature raise some degrees higher then set. Nothin to worry about.

Here is a screenshot:
![temp](https://cloud.githubusercontent.com/assets/18073299/22660815/31e54f26-eca3-11e6-91ab-5ebe4bc55849.PNG)

But what is realy strange, that I noticed **this** behavior while I was printing a cube:
![t defekt](https://cloud.githubusercontent.com/assets/18073299/22660858/5d31a0da-eca3-11e6-97f0-2b5b6ad2fde5.PNG)

So I checked the hardware and also replaced a new thermistor.

Here is my [Configuration.h](http://pastebin.com/URaRfwuf) (pastebin)

What else information do you need to solve this?

**Comment by Blue-Marlin** [comment]
If you have activated `ADVANCE` or `LIN_ADVANCE` please deactivate both, retest and report back.

**Comment by TylonHH** [comment]
I double checked
`//#define ADVANCE`
`//#define LIN_ADVANCE`
and reuploaded the firmware.

Problem still persist

**Comment by thinkyhead** [comment]
Have you done a new `M303 U1` to re-tune PID for the new firmware (followed by `M500`)?

**Comment by Blue-Marlin** [comment]
@thinkyhead 
Have you had a look on the graphs? This can not be reasoned by the PID-values. Jumps by 50 and more degrees can only come from the measurement. Temperature systems are much too inert for these jumps.

**Comment by TylonHH** [comment]
No I didn't. Because the heatbed makes this trouble and the bed is not PID controlled.
But I run your commands and saved them.

**Comment by TylonHH** [comment]
@thinkyhead so you guess its an hardware problem? Maybe some electrical inductive stuff?

**Comment by 3dGriff** [comment]
@TylonHH I will second @Blue-Marlin on this one.  Although I'm not sure what your heat bed is made of, given its (likely) thermal mass, drops like these are most likely a hardware issue.  Obviously check the connections, but more importantly check the thermistor connection closest to the resistor, they have a tendency to degrade based on our experience.  

On another note, in the past we have seen strange results like this in the past with RAMPS boards.  The issue occurred due to heat on the board caused by running both 12v power and USB.  If you can, let the machine sit (to cool down) and then attempt to heat the bed without the USB connected.  This is just speculation.

**Comment by mauriciohvargas** [comment]
Hi, i have the same problem, so thinking it was a hardware problem i uploaded the firmware in other board (arduino Mega with RAMPS) and plugged just other new thermistors (no stepper motors, nothing else in fact) and the readings were the same using the same test gcode. This makes me think that there is no hardware problem.
After some tries with the firmware i just commented the #define COREXY line and there were no weird readings. So i configured the printer firmware with commented #define COREXY line AND UNCOMMENTED #define COREYX and it worked. I am sure there is a bug with the COREXY code.

**Comment by TylonHH** [comment]
@3dGriff 
> and then attempt to heat the bed without the USB connected.

But without the USB cable I can't see the graph vie S3D ;-)

EDIT: I also tried some ferrit rings around the sensors, but no luck.

**Comment by 3dGriff** [comment]
@TylonHH it seems that @mauriciohvargas has seen a connection the #define COREXY.  I tried to replicate this but wasn't able to reproduce the results.  I can try again when I get back to the shop. Have you attempted what @mauriciohvargas suggested to verify the connection?

**Comment by thinkyhead** [comment]
> I am sure there is a bug with the COREXY code.

Possibly. But don't be too sure. Remember the maxim: correlation does not necessarily imply causation. Note that `COREXY` produces precisely the same code as `COREYX`, with only one value negated.

**Comment by billyd60** [comment]
I get a similar kind of behavior as well with rcbugfix but only when I cancel a print from the lcd panel and restart a print again without a power cycle to the printer board. If I don't cancel a print from the lcd panel my temps are rock solid. ps I always print from SD card never USB.

**Comment by Roxy-3D** [comment]
> I am sure there is a bug with the COREXY code.

I'm sure there is a problem somewhere...  But I can tell you this:   When I brought up the UBL Bed Leveling system on a normal Prusa i3, absolutely nothing had to be done to make it work on Core-XY machines.   That is simply amazing.   That is only one data point, but I'm of the opinion the Core-XY stuff is 'Correct'.

**Comment by mauriciohvargas** [comment]
It has to be a bug, because i switch to 1.1.0 RC6 with the same configuration and there is no problem. The only thing observed is that when the print starts and it's homing, the bed temperature reading drops to 0 and then it rices to normal.
When going again to RC7 or RC8 the problem reappears.

**Comment by thinkyhead** [comment]
#5829 might address this.

**Comment by Anxles** [comment]
I see there are some doubts about is it a software and hardware problem. By no meens I am a programmer, I even struggle a little with configuration.h to understand all options. BUT I have same problem with RC8, I didn't try G29 but I have problem sa soon as I start the print. I have CORE XY enabled, printer of my own construction on Megatronics V2 board. everything was OK before but I had to make dual Z stepper for torque on Z axis, and I had to update Marlin for that as the old one didn't support that option for my board, I changed all the relevant settings, uploaded firmware, tested all the axis lcd etc (everything looked fine) the problem started when I tried to make a test print. As soon as it started BED temperature became ERRATIC jumping bewteen 400+ C and some LOW (I didn't notice how much) the jumps were sudden a few hunder degrees )
Long story short I checked the thermistor and connections, Changed to new thermistor and result is the same. After looking here and found people with the same problem, I configured RC6 to same parameters as RC8 and so far for few hours it's printing but still I can see some small and very short temp jumps (I have bed 40x40cm 10mm thick so thermal capacity is huge - by no meens possible any sudden temp, jumps).
Hope this will be resolved soon as options available at RC8 are very interesting and somehow I dont like Repetier firmware :)
